Coordination meeting 2022-09-15 (#1251)
HSF/hsf.github.io: organization/_posts/2022/2022-09-15-coordination.md

### LHCC

Slides presented at the LHCC are attached to the [agenda](https://indico.cern.ch/event/1096598/).

LHCC Feedback:

#### News from many fronts (a selection)

- Event Generators: covering theoretical, experimental and computing aspects (possible role of AI/ML)
- Detector simulation: Fast Calorimeter Simulation Challenge; Recent developments and experience with DD4hep
- Simulation: Geant4 11.1-Beta released, new VecGeom release v1.2.0 (enhanced CUDA support, C++17 default std)
- ROOT: 6.26/06 released, current recommended release. All experiments agree to use 6.24 for production, at least for the initial part of Run 3. Simplifies maintenance for ROOT.
- CernVM File System: Version 2.10 feature complete, pre-release available

#### Personnel turn-over

- ROOT: Significant challenges due to departure / time reduction of key people, for instance end of fellowship for inventor of RDataFrame, departure of CernVM LD affecting RNTuple development -> CERN is willing to hire new collaborators
- EU grant SYCLOPS as partner (~ 450 k€ for ROOT) - RDataFrame offloading (GPU and possibly other accelerators)
- CernVM File System: a talented staff left to industry -> position is reopened

> - Collaboration with private companies: one fellow funded from Jumptrading

Reviewer: ```suggestion
- Collaboration with private companies: one fellow funded from Jump Trading
```

#### Training & attracting young talents

- Training: Mathplotlib, Software carpentry, hackatons, Advanced C++…, Geant4 (beginners, advanced)
- Google Summer of Code:
  - HSF is participating in the program as a GSoC umbrella organization (CERN-HSF)
  - Proposition of projects on common SW (26 participating organisation)
  - 2022: Google granted a quota of 27 students ; all passed mid-term evaluation (projects to be finished in Sept.)

The WLCG Software Report for the RRB (2022-H2) is available [here](https://docs.google.com/document/d/1WbwmWYe1bp_TJMwTfOdMlm3ecAmlpuH51mgRnhqU9nQ/edit?usp=sharing). It will be submitted Friday 16 September, so please check now if there's anything to add or correct.

### Twitter

Twitter goes at speed for the HSF!

> We have a new training dedicated twitter: [@hsftraining](https://twitter.com/HsfTraining). We encourage you to follow this and the main HSF twitter, [@hepsoftfound](https://twitter.com/hepsoftfound).

Reviewer: ```suggestion
We have a new training dedicated twitter account: [@hsftraining](https://twitter.com/HsfTraining). We encourage you to follow this and the main HSF twitter, [@hepsoftfound](https://twitter.com/hepsoftfound).
```

### PyHEP

- PyHEP workshop is ongoing, with more than 1000 people registered
  - Peak attendance on Monday was more than 200
  - Now at to a steady 75 per session
  - Two social events held on *Virtually Green*
  - Hackashop running this evening and tomorrow morning

### Software Tools and Packaging

- Finalizing a series of Spack discussions with LCG, CMS, LHCb and Spack developers
  - We discussed Spack training and think this would be a good idea, both for users and developers
  - There is a view/environment tutorial (sort of) here: <https://spack-tutorial.readthedocs.io/en/latest/tutorial_environments.html>
- mail discussion regarding conda with a PUNCH developer, possible meeting
- Meeting on ACTS build time optimizations during the summer was sparsely attended, repeat discussion at a future meeting

### Event Generators

- Very nice talk from Anja Butter (and well attended meeting) on ML for generators at [last meeting](https://indico.cern.ch/event/1180220/)
- Next meeting: Electron Ion Collider and Nuclear Physics Generators by Markus D., Oct, 6th. ([Agenda](https://indico.cern.ch/event/1200496/)).

> - Discussing next topics for the meetings until the end of the year. May follow up on ML for generators.

Reviewer: ```suggestion
- Discussing the next topics for the meetings until the end of the year. May follow up on ML for generators.
```
